extensions/src/main/java/org/pentaho/platform/plugin/action/ActionInvokerSystemListener.java

```diff
@@ -77,21 +79,24 @@ public boolean startup( IPentahoSession session ) {
     }
 
     for ( File file : files ) {
+      Payload payload = null;
       logger.info( "Attempting to read data from " + file.getAbsolutePath() );
       try {
         FileInputStream fileInputStream = new FileInputStream( file );
         String encoded = IOUtils.toString( fileInputStream );
         fileInputStream.close();
-        Payload payload = new Payload( encoded, LocaleHelper.UTF_8 );
+        payload = new Payload( encoded, LocaleHelper.UTF_8 );
         logger.info( "Issuing Work Item request" );
         try {
           IActionInvokeStatus status = issueRequest( payload );
         } catch ( Exception e ) {
-          //send status to chronos
+          // TODO: send status to chronos
```

Comment:
Already taken care of indirectly: issueRequest > payload.issueRequest >  getActionInvoker().invokeAction( action, actionUser, actionMap ) - any failures would be reported inside invokeAction